deploy/crd/aquasecurity.github.io_clustercompliancereports.yaml

```diff
@@ -0,0 +1,153 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.0
+  creationTimestamp: null
+  name: clustercompliancereports.aquasecurity.github.io
+spec:
+  group: aquasecurity.github.io
+  names:
+    kind: ClusterComplianceReport
+    listKind: ClusterComplianceReportList
+    plural: clustercompliancereports
+    singular: clustercompliancereport
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: ClusterComplianceReport is a specification for the ClusterComplianceReport
+          resource.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: ReportSpec represent the compliance specification
+            properties:
+              controls:
+                items:
+                  description: Control represent the cps controls data and mapping
+                    checks
+                  properties:
+                    defaultStatus:
+                      type: string
+                    description:
+                      type: string
+                    id:
+                      type: string
+                    kinds:
+                      items:
+                        type: string
+                      type: array
+                    mapping:
+                      description: Mapping represent the scanner who perform the control
+                        check
+                      properties:
+                        checks:
+                          items:
+                            description: SpecCheck represent the scanner who perform
+                              the control check
+                            properties:
+                              id:
+                                type: string
+                            required:
+                            - id
+                            type: object
+                          type: array
+                        scanner:
+                          type: string
```

Comment:
and here : 
  scanner:
                            type: string
                            pattern: '^config-audit$'